packages/components/codemods/migrateInformationTileMoodToVariant/transformInformationTileMoodToVariant.ts

```diff
@@ -0,0 +1,22 @@
+import type { InformationTileProps } from "~components/Tile"
+import { migrateStringProp } from "../utils/migrateStringProp"
+
+const OLD_PROP_NAME = "mood"
+const NEW_PROP_NAME = "variant"
+
+const getNewVariantValue = (
+  oldValue: Exclude<InformationTileProps[typeof OLD_PROP_NAME], undefined>
+): Exclude<InformationTileProps[typeof NEW_PROP_NAME], undefined> => {
+  switch (oldValue) {
+    case "prominent":
+      return "expert-advice"
+    default:
+      return "default"
```

Comment:
Shouldn't the prop be removed to go to the default?

Reply:
Hmm didn't consider that. I was thinking that the codemods are always explicit 🤔 Let's chat about it

Reply:
As discussed, we'll go with explicit definition so that we lessen possible confusion for consumers :)